Zend/tests/enum/scalar-cases-string.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+String scalar enums with can list cases
+--FILE--
+<?php
+
+enum Suit: string {
+    case Hearts = 'H';
+    case Diamonds = 'D';
+    case Clubs = 'C';
+    case Spades = 'S';
+}
+
+var_dump(Suit::cases());
+
+?>
+--EXPECT--
+array(4) {
+  ["H"]=>
+  enum(Suit::Hearts)
+  ["D"]=>
+  enum(Suit::Diamonds)
+  ["C"]=>
+  enum(Suit::Clubs)
+  ["S"]=>
+  enum(Suit::Spades)
+}
```

Comment:
The RFC states that cases() doesn't include keys. Just want to check whether that's the intended final behavior or not.

Reply:
I probably forgot to update the implementation here.